src/mantine-rte/src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -44,10 +45,10 @@ export interface RichTextEditorProps
   extends DefaultProps<RichTextEditorStylesNames>,
     Omit<React.ComponentPropsWithoutRef<'div'>, 'onChange'> {
   /** HTML content, value not forced as quill works in uncontrolled mode */
-  value: string;
+  value: string | Delta;
```

Comment:
We cannot change value type, it will be breaking. Adding types to `onChange` is fine.

Reply:
But this is input value. This is just extend types which you can provide to RTE component. Also, we leave `string` type

Reply:
Ah, yes, that's true, sorry, I was thinking of something else